requirements.txt

```diff
@@ -2,6 +2,7 @@ adal==0.4.3
 applicationinsights==0.10.0
 argcomplete==1.3.0
 colorama==0.3.7
+coverage==4.2
```

Comment:
And I don't think this `requirement.txt` should impact the user dependencies which are defined in the `setup.py` as far as I know. @derekbekoe